libbpf-tools/filelife.bpf.c

```diff
@@ -6,6 +6,9 @@
 #include <bpf/bpf_tracing.h>
 #include "filelife.h"
 
+// linux: include/linux/fs.h
+#define FMODE_CREATE	0x100000
```

Comment:
No C++ style comments.

FMODE_CREATED ?

Reply:
Like `fs/namei.c` say:

```c
    /* Negative dentry, just create the file */                                 
    if (!dentry->d_inode && (open_flag & O_CREAT)) {                            
        file->f_mode |= FMODE_CREATED;                                                                                      
        audit_inode_child(dir_inode, dentry, AUDIT_TYPE_CHILD_CREATE);          
        if (!dir_inode->i_op->create) {                                         
            error = -EACCES;                                                    
            goto out_dput;                                                          
        }
```

open(..., O_CREAT, ...) set FMODE_CREATED, that is create new file. so we need FMODE_CREATE, but can't parse:

```c
linux/fs.h:161:#define FMODE_CREATED		((__force fmode_t)0x100000)
```

So i just add `#define FMODE_CREATE	0x100000` in front of `filelife.bpf.c`

Reply:
`FMODE_CREATE` -> `FMODE_CREATED`

Reply:
Resolved in the new force commit.